charts/managed-serviceaccount/templates/addontemplate.yaml

```diff
@@ -8,6 +8,12 @@ spec:
   agentSpec:
     workload:
       manifests:
+      - apiVersion: v1
```

Comment:
add a comment here indicating this is to handle upgrade, and should be removed in the later release

Reply:
PTAL